.travis.yml

```diff
@@ -1,19 +1,23 @@
 language: php
 
-# Setting sudo access to false will let Travis CI use containers rather than
-# VMs to run the tests. For more details see:
-# - http://docs.travis-ci.com/user/workers/container-based-infrastructure/
-# - http://docs.travis-ci.com/user/workers/standard-infrastructure/
 sudo: false
 
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
+matrix:
+    include:
+        - php: 5.3
+          dist: precise
+    fast_finish: true
+
+dist: trusty
```

Comment:
I'm likely missing something but, why are precise and trusty being specified?

Reply:
`trusty` does not support php 5.3, so `precise` is necessary.

`trusty` was not the default build platform on travis ci at the time of the commit